R/bootstrap-functions.R

```diff
@@ -54,7 +54,7 @@
 #' @rdname bootstrap
 #' @importFrom parallel mclapply
 #' @export
-bootstrap <- function(x, fun, n=1000L, mc.cores=getOption("mc.cores", 2L)) {
+bootstrap <- function(x, fun, n=1000L, mc.cores=1) {
```

Comment:
I would prefer the following:
```suggestion
bootstrap <- function(x, fun, n=1000L, mc.cores=getOption("mc.cores", 1L)) {
```
Because then I could set `options(mc.cores=2)` and don't have to take care about the argument anymore.